app/controllers/tweet_controller.rb

```diff
@@ -0,0 +1,80 @@
+class TweetController < ApplicationController
+
+  get '/tweets' do
+    if logged_in?   
+	  erb :'tweets/index'
+	else
+	  redirect to '/login'
+	end  
+  end
+
+  get '/tweets/new' do
+  	if logged_in?
+  	  erb :'tweets/new'
+  	else
+  		redirect to '/login'
+  	end  
+  end
+
+  post '/tweets' do
+    @user = User.find_by_id(session[:user_id])
+    if params[:content] == ""
+    	redirect to '/tweets/new'
+    else
+  	    @user.tweets.create(content: params[:content])
+  	    @user.save
+  	    redirect to '/tweets'
+    end
+  end
+
+  get '/tweets/:id' do
+    if logged_in?
+	    @tweet = Tweet.find_by_id(params[:id])
+	  	erb :'tweets/show'
+	  else
+	  	redirect '/login'
+	  end
+  end
+
+  get '/tweets/:id/edit' do
+  	if logged_in?
+  		@tweet = Tweet.find_by_id(params[:id])
```

Comment:
Shouldn't you verify a tweet belongs to the `current_user` (and not just that any user is logged in) before allowing them to edit?